Libplanet.Tests/Net/SwarmTest.Broadcast.cs

```diff
@@ -631,19 +634,23 @@ public async Task BroadcastBlockWithSkip()
 
                 await BootstrapAsync(receiverSwarm, minerSwarm.AsPeer);
 
+                HashAlgorithmType hashAlgo1 = HashAlgorithmType.Of<SHA256>();
```

Comment:
Do we need `hashAlgo1` (and `hashAlgo2`)?

Reply:
No, they are no more needed.  I'll remove them.

Reply:
Amended!